src/pages/CreateProtocolPage.jsx

```diff
@@ -222,15 +246,35 @@ function CreateProtocolPage(props) {
                                     onChange={(event) => setDescription(event.target.value)}
                                 ></textarea>
                             </div>
-                            {inputs.map((input, index) => (
-                                <CreateTextBoxInput
-                                    key={index}
-                                    index={index}
-                                    inputState={input}
-                                    onTextBoxChange={handleTextBoxChange}
-                                    onTextBoxRemove={() => handleTextBoxRemove(index)}
-                                />
-                            ))}
+                            {inputs.map((input, index) => {
+                                switch (input.id) {
+                                    case 1:
+                                        item = (
+                                            <CreateTextBoxInput
+                                                key={index}
+                                                index={index}
+                                                inputState={input}
+                                                onTextBoxChange={handleTextBoxChange}
+                                                onTextBoxRemove={() => handleTextBoxRemove(index)}
+                                            />
+                                        );
+                                        break;
+                                    case 3:
```

Comment:
```suggestion
                                switch (input.type) {
                                    case 0:
                                        item = (
                                            <CreateTextBoxInput
                                                key={index}
                                                index={index}
                                                inputState={input}
                                                onTextBoxChange={handleTextBoxChange}
                                                onTextBoxRemove={() => handleTextBoxRemove(index)}
                                            />
                                        );
                                        break;
                                    case 2:
```
- Mesma justificativa anterior;